feature-find-meal/src/main/java/com/kinandcarta/feature/find/meal/view/FindMealFragment.kt

```diff
@@ -1,115 +1,102 @@
-        mealsRecyclerView.adapter = mealsAdapter
-        val swipeRefreshLayout: SwipeRefreshLayout = requireView().findViewById(R.id.swipeContainer)
-        swipeRefreshLayout.setOnRefreshListener {
-            swipeRefreshLayout.isRefreshing = false
-            viewModel.updateMeals()
-        }
+    fun setupUI() {
+        viewModel.startUpdatingLocation()
     }
 
-    private fun onStateChanged(state: FindMealViewModel.State) {
+    private fun onStateChanged(state: DisplayMealsViewModel.State) {
         when (state) {
-            FindMealViewModel.State.LoadingMeals ->
+            DisplayMealsViewModel.State.LoadingMeals ->
                 progressBarView.isVisible = true
-            is FindMealViewModel.State.LoadedMeals ->
+            is DisplayMealsViewModel.State.LoadedMeals ->
                 onLoadedMeals(state)
-            is FindMealViewModel.State.ReservedMeal ->
+            is DisplayMealsViewModel.State.ReservedMeal ->
                 onReservedMeal(state)
-            is FindMealViewModel.State.MealUnavailable ->
+            is DisplayMealsViewModel.State.MealUnavailable ->
                 onMealUnavailable(state)
-            is FindMealViewModel.State.Failed ->
+            is DisplayMealsViewModel.State.Failed ->
                 onFailure(state.failure)
         }
     }
 
-    private fun onLoadedMeals(state: FindMealViewModel.State.LoadedMeals) {
+    private fun onLoadedMeals(state: DisplayMealsViewModel.State.LoadedMeals) {
         progressBarView.isVisible = false
         mealsAdapter.submitList(state.meals)
+        Log.i("Find meal fragment", "onLoadedMeals ${state.meals}")
     }
 
-    private fun onReservedMeal(state: FindMealViewModel.State.ReservedMeal) {
+    private fun onReservedMeal(state: DisplayMealsViewModel.State.ReservedMeal) {
         showToast("Your meal reservation code is ${state.code}")
     }
 
-    private fun onMealUnavailable(state: FindMealViewModel.State.MealUnavailable) {
+    private fun onMealUnavailable(state: DisplayMealsViewModel.State.MealUnavailable) {
         showToast("Unfortunately this meal is unavailable")
     }
 
-    private fun onFailure(failure: FindMealViewModel.Failure) {
+    private fun onFailure(failure: DisplayMealsViewModel.Failure) {
         when (failure) {
-            is FindMealViewModel.Failure.LoadingMealsFailed -> {
+            is DisplayMealsViewModel.Failure.LoadingMealsFailed -> {
                 progressBarView.isVisible = false
                 showToast(failure.localizedMessage ?: "An unexpected error occurred loading meals")
             }
-            is FindMealViewModel.Failure.ReserveAMealFailed -> {
-                showToast(failure.localizedMessage ?: "An unexpected error occurred reserving a meal")
+            is DisplayMealsViewModel.Failure.ReserveAMealFailed -> {
+                showToast(
+                    failure.localizedMessage ?: "An unexpected error occurred reserving a meal"
```

Comment:
Are you hardcoding strings for now instead of putting them in `strings.xml`?

Reply:
Ah, this must have been missed, will move across!